test/integration/scheduler/scheduler_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp"
+
```

Comment:
not sure what the convention is, but it might be more common to have it in the next import block